clients/v1_shell/table_model.go

```diff
@@ -171,9 +171,12 @@ func (m Model) Init() tea.Cmd {
 	if len(m.tags) > 0 {
 		fmt.Print("Tags: ")
 		for key, val := range m.tags {
-			if key == "" || val == "" {
+			if key == "" {
 				continue
 			}
+			if val == "" {
+				val = " ----- "
```

Comment:
What do you think about using something like <empty>, similar to <nil>?

Reply:
I'm not sure I understand the question.  Are you suggesting using a blank line or a comma?  The field at this point is mostly presentational.  I think it's important to have a pseudo-graphic signifier that suggests the key was tried and that unambiguously it returned an empty value.  A blank line as a signifier leaves a far more open set of possible interpretations. 

For the query `SHOW FIELD KEY CARDINALITY FROM test.autogen.iota GROUP BY id`

Currently
```
Interactive Table View (press q to exit mode, shift+up/down to navigate tables):
Name: iota
Tags: id= ----- 
┏━━━━━━━┳━━━━━━━━━━━━━━┓
┃ index ┃    count     ┃
┣━━━━━━━╋━━━━━━━━━━━━━━┫
┃      1┃  1.0000000000┃
┣━━━━━━━┻━━━━━━━━━━━━━━┫
┃    2 Columns, 1 Rows,┃
┃              Page 1/1┃
┃  Table 1/1, Statement┃
┃                   1/1┃
┗━━━━━━━━━━━━━━━━━━━━━━┛
```
Proposed?
```
Interactive Table View (press q to exit mode, shift+up/down to navigate tables):
Name: iota
Tags: id=
┏━━━━━━━┳━━━━━━━━━━━━━━┓
┃ index ┃    count     ┃
┣━━━━━━━╋━━━━━━━━━━━━━━┫
┃      1┃  1.0000000000┃
┣━━━━━━━┻━━━━━━━━━━━━━━┫
┃    2 Columns, 1 Rows,┃
┃              Page 1/1┃
┃  Table 1/1, Statement┃
┃                   1/1┃
┗━━━━━━━━━━━━━━━━━━━━━━┛
```

Reply:
OK following Vlasta's recommendation empty tag values now render like this

```
Interactive Table View (press q to exit mode, shift+up/down to navigate tables):
Name: iota
Tags: id= <nil> 
┏━━━━━━━┳━━━━━━━━━━━━━━┓
┃ index ┃    count     ┃
┣━━━━━━━╋━━━━━━━━━━━━━━┫
┃      1┃  1.0000000000┃
┣━━━━━━━┻━━━━━━━━━━━━━━┫
┃    2 Columns, 1 Rows,┃
┃              Page 1/1┃
┃  Table 1/1, Statement┃
┃                   1/1┃
┗━━━━━━━━━━━━━━━━━━━━━━┛
```